src/KrakenBattle.cpp

```diff
@@ -160,7 +160,7 @@ namespace KrakenBattle {
          case ProgressType::Stop : textType = "ProgressType::Stop"; break;
          case ProgressType::Continue : textType = "ProgressType::Continue"; break;
          default:
-            LOG(FATAL) << "Unknown Send::Type: " << static_cast<int>(type);
+            LOG(FATAL) << "Unknown ProgressType: " << static_cast<int>(type);
```

Comment:
no test exist for this it was part of the previous refactoring. I consider this tech debt